packages/clerk-js/src/core/clerk.ts

```diff
@@ -948,7 +952,36 @@ export default class Clerk implements ClerkInterface {
     this.#componentControls?.updateProps(props);
   };
 
-  #loadInStandardBrowser = async (): Promise<void> => {
+  #redirectToSyncWithPrimary = () => {
+    const q = new URLSearchParams({
+      redirect_url: `${window.location.href}?synced=true`,
```

Comment:
❓ Is this the function that initiates the syncing process? If yes, it's a bit confusing that the query string is `synced=true` at this point. Am I missing something?

Reply:
Better naming for the syncing process was introduced [here](https://github.com/clerkinc/javascript/pull/727).

But to give you more context: There we are constructing the return URL which will point back to our app. Once you came back if `synced=true` the syncing process will not fire again (avoiding an infinite loop)

Until now this is handled inside clerk-js, I believe this could also be handled by the `/link` endpoint, but I don't believe this is necessary